### ISSUE TITLE
Running the template causes a bunch of dependency downgrade errors on build.

### ISSUE THREAD

**Issue by PartemImperium** [net-daemon/netdaemon-app-template#64]
The template out of the box does not build due to "error NU1605: Detected package downgrade: Microsoft.Extensions.Hosting from 6.0.1 to 6.0.0. Reference the package directly from the project to select a different version. " (part of the very long build output). The fix is to just update the "Serilog.AspNetCore" from 4.1.0 -> 5.0.0 and "Microsoft.Extensions.Hosting" from 6.0.0 -> 6.0.1

**Comment by PartemImperium** [comment]
If there is a expected format for Issues that I didnt see feel free to point me to it and I can make sure to adhere to such,

**Comment by helto4real** [comment]
Thanks for reporting, please upgrade the packages manually in the project file and I will fix this tonight.

**Comment by PartemImperium** [comment]
Actually, looking at https://github.com/net-daemon/netdaemon-app-template/blob/main/daemonapp.csproj#L27-L28 it already has the correct versions... 

The version of the template dotnet new used is 22.13.0. I made sure to run "dotnet new --install JoySoftware.NetDaemon.Templates.Project" before generating and that is the newest version that it gave me. Looking at nuget that is the most recent version and it was added 2 days ago. What is the branching strategy for this repo/ is the nuget package made from the main branch?
